src/languages/en.json

```diff
@@ -24,15 +24,16 @@
     "invoice.amount": "Amount",
     "invoice.notes": "Notes",
     "invoice.date": "Date",
-    "invoice.cutomse.name": "Customer",
+    "invoice.customer.name": "Customer",
     "invoice.paid.status": "Paid Status",
     "invoice.amount.due": "Amount Due",
     "invoice.change": "change",
-    "invoice.add.on.item": "Add an item",
+    "invoice.add.an.item": "Add an item",
     "invoice.total.amount": "Total Amount",
     "invoice.model": "Model",
     "no.records": "There are no records available",
-    "invoice.add.customer.details": "Add  Customer details",
+    "invoice.select.customer": "Select Customer",
```

Comment:
same here fix everywhere invoice.selectCustomer

Reply:
will do the changes in all possible keys

Reply:
@gautism1 please do fixes in this branch